week-3/22-checkout/index.html

```diff
@@ -37,8 +37,12 @@ <h1 class="shopping-basket__title">Your Shopping Basket</h1>
         <div class="checkout">
           <h2 class="checkout__title">Checkout</h2>
           <form method="GET" id="checkout-form" class="checkout__form">
-            <!-- Add the form fields here -->
-
+<label> for="customer-name">customer-name</label>
+<input id="customer-name" type="text" name="customer-name">
+<label for="email">customer-email</label>
+<input id="email" type="email" name="customer-email">
+<label for="tel">customer-phone</label> 
```

Comment:
The `for` attribute must match the `id` attribute of the element it corresponds to. The `id` isn't `tel` in this case.